tests/integration/diff_test.go

```diff
@@ -461,7 +461,7 @@ Summary:
 }
 
 `
-	expectedOutputPluginUpdateNoChange = `Summary:
+	expectedOutputNoChange = `Summary:
```

Comment:
Changed variable name here so that we could reuse it. This wasn't specific to plugins anyway.